app/components/Contacts/ContactsPanel/ContactsPanel.jsx

```diff
@@ -158,7 +158,7 @@ export default class ContactsPanel extends React.Component<Props, State> {
     this.setState({ sorting: option })
   }
 
-  handleEdit = (name: string) => () => {
+  handleEdit = (name: string) => {
     this.props.history.push(`/contacts/edit/${encodeURIComponent(name)}`)
   }
 
```

Comment:
While you are here do you mind removing handleDelete (its no longer being used in this component as delete is an option in the edit view now)

Reply:
you can just remove it after we merge this PR. It looks good